### PR TITLE
Update default plugin installation versions

### LINKED ISSUE #704: Plugin depends on
### Describe the bug

jenkins-0 pod will not start up. Keep getting back-off error. It's happening all of a sudden. 

### Version of Helm and Kubernetes

```markdown
- Helm: `version.BuildInfo{Version:"v3.9.3", GitCommit:"414ff28d4029ae8c8b05d62aa06c7fe3dee2bc58", GitTreeState:"clean", GoVersion:"go1.17.13"}`
- Kubernetes: `Client Version: version.Info{Major:"1", Minor:"25", GitVersion:"v1.25.0", GitCommit:"a866cbe2e5bbaa01cfd5e969aa3e033f3282a8a2", GitTreeState:"clean", BuildDate:"2022-08-23T17:44:59Z", GoVersion:"go1.19", Compiler:"gc", Platform:"windows/amd64"}
Kustomize Version: v4.5.7
Server Version: version.Info{Major:"1", Minor:"24", GitVersion:"v1.24.4+k3s1", GitCommit:"c3f830e9b9ed8a4d9d0e2aa663b4591b923a296e", GitTreeState:"clean", BuildDate:"2022-08-25T03:45:26Z", GoVersion:"go1.18.1", Compiler:"gc", Platform:"linux/amd64"}`
```

### Chart version

from 4.1.8 to 4.2.2 

### What happened?

```bash
1. helm install would time out, and I keep getting this...
`$ kubectl get pods jenkins-0
NAME        READY   STATUS                  RESTARTS   AGE
jenkins-0   0/2     Init:CrashLoopBackOff   6          1m11s`

2. logs show up for init container:
`Will install new plugin workflow-api 1192.v2d0deb_19d212
io.jenkins.tools.pluginmanager.impl.AggregatePluginPrerequisitesNotMetException: Multiple plugin prerequisites not met:
Plugin workflow-aggregator:581.v0c46fa_697ffd (via pipeline-model-definition:2.2114.v2654ca_721309->git-client:3.12.0) depends on configuration-as-code:1512.vb_79d418d5fc8, but there is an older 
version defined on the top level - configuration-as-code:1429.v09b_044a_c93de,
Plugin git:4.11.3 (via git-client:3.12.0) depends on configuration-as-code:1512.vb_79d418d5fc8, but there is an older version defined on the top level - configuration-as-code:1429.v09b_044a_c93de        at io.jenkins.tools.pluginmanager.impl.PluginManager.start(PluginManager.java:240)
        at io.jenkins.tools.pluginmanager.impl.PluginManager.start(PluginManager.java:189)
        at io.jenkins.tools.pluginmanager.cli.Main.main(Main.java:52)
        Suppressed: io.jenkins.tools.pluginmanager.impl.PluginDependencyException: Plugin workflow-aggregator:581.v0c46fa_697ffd (via pipeline-model-definition:2.2114.v2654ca_721309->git-client:3.12.0) depends on configuration-as-code:1512.vb_79d418d5fc8, but there is an older version defined on the top level - configuration-as-code:1429.v09b_044a_c93de
                at io.jenkins.tools.pluginmanager.impl.PluginManager.resolveRecursiveDependencies(PluginManager.java:1125)
                at io.jenkins.tools.pluginmanager.impl.PluginManager.findPluginsAndDependencies(PluginManager.java:692)
                at io.jenkins.tools.pluginmanager.impl.PluginManager.start(PluginManager.java:232)
                ... 2 more
        Suppressed: io.jenkins.tools.pluginmanager.impl.PluginDependencyException: Plugin git:4.11.3 (via git-client:3.12.0) depends on configuration-as-code:1512.vb_79d418d5fc8, but there is an 
older version defined on the top level - configuration-as-code:1429.v09b_044a_c93de
                at io.jenkins.tools.pluginmanager.impl.PluginManager.resolveRecursiveDependencies(PluginManager.java:1125)
                at io.jenkins.tools.pluginmanager.impl.PluginManager.findPluginsAndDependencies(PluginManager.java:692)
                at io.jenkins.tools.pluginmanager.impl.PluginManager.start(PluginManager.java:232)
                ... 2 more
Multiple plugin prerequisites not met:
Plugin workflow-aggregator:581.v0c46fa_697ffd (via pipeline-model-definition:2.2114.v2654ca_721309->git-client:3.12.0) depends on configuration-as-code:1512.vb_79d418d5fc8, but there is an older 
version defined on the top level - configuration-as-code:1429.v09b_044a_c93de,
Plugin git:4.11.3 (via git-client:3.12.0) depends on configuration-as-code:1512.vb_79d418d5fc8, but there is an older version defined on the top level - configuration-as-code:1429.v09b_044a_c93de`
```

### What you expected to happen?

Could I update for the conflicting plugin version without having to create a custom image?

### How to reproduce it

_No response_

### Anything else we need to know?

I dont apply values.yaml in this tests

### PR DESCRIPTION
**Pull request by pedringcoding**
Fixing plugin version matrix to some default installs

fixes #704, will close that issue when PR gets merged

### CODE REVIEW

File: charts/jenkins/values.yaml [hunk 1]
```diff
@@ -241,7 +241,7 @@ controller:
     - kubernetes:3600.v144b_cd192ca_a_
     - workflow-aggregator:581.v0c46fa_697ffd
     - git:4.11.3
```
Comment (lemeurherve) [inline charts/jenkins/values.yaml:243]:
```suggestion
    - kubernetes:3706.vdfb_d599579f3
    - workflow-aggregator:590.v6a_d052e5a_a_b_5
    - git:4.11.5
```
Updating all installed plugins

**Review by lemeurherve — DISMISSED** [review]
Good work!

**Comment by lemeurherve** [comment]
Would supersede https://github.com/jenkinsci/helm-charts/pull/690

**Comment by timja** [comment]
(and fix DCO)

**Comment by pedringcoding** [comment]
I have unified all the commits to avoid some smelly messages sheet to doesn't too much traces, but  at moment, for this change, not permitted complete a GitHub actions... Could you help me? @timja

**Comment by timja** [comment]
> I have unified all the commits to avoid some smelly messages sheet to doesn't too much traces, but at moment, for this change, not permitted complete a GitHub actions... Could you help me? @timja

502 bad gateway from somewhere, I've reran the build.

**Review by timja — APPROVED** [review]
thanks!

**Pull request merged**